tests/pyramid/test_pyramid.py

```diff
@@ -0,0 +1,73 @@
+from typing import Tuple
+from pathlib import Path
+
+import numpy as np
+
+from ome_zarr.io import parse_url
+from ome_zarr.reader import Reader, Multiscales
+
+from iohub.ngff import open_ome_zarr, Position
+from iohub.ngff_meta import TransformationMeta
+from iohub.pyramid import initialize_pyramid
+
+
+def _mock_fov(
+    tmp_path: Path,
+    shape: Tuple[int, ...],
+    scale: Tuple[float, float, float],
+) -> Position:
+
+    ds_path = tmp_path / "ds.zarr"
+    channels = [str(i) for i in range(shape[1])]
+
+    fov = open_ome_zarr(ds_path, layout="fov", mode="a", channel_names=channels)
+    transform = [
+        TransformationMeta(
+            type="scale",
+            scale=(1, 1) + scale,
+        )
+    ]
+
+    if "0" not in fov.array_keys():
+        fov.create_zeros(
+            "0",
+            shape=shape,
+            dtype=np.uint16,
+            chunks=(1, 1, 128) + shape[-2:],
+            transform=transform,
+        )
+    
+    return fov
+
+
+def test_pyramid(tmp_path: Path) -> None:
```

Comment:
I think this is a good place to fuzz the test. @JoOkuma Would you mind if I push some changes here?

Reply:
Go a head